Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic.py

```diff
@@ -0,0 +1,321 @@
+        parser.add_argument('-out', '--output-path', type=pathlib.Path, required=True, help='Path that describes the final resulting Release Directory path location.')
+        parser.add_argument('-cfg', '--config', type=str, default='profile', choices=['release', 'profile'],
+                                help='The CMake build configuration to use when building project binaries. If tool binaries are built with this script, they will use profile mode.')
+        parser.add_argument('-a', '--archive-output',
+                                type=str,
+                                help="Option to create a compressed archive the output. "
+                                "Specify the format of archive to create from the output directory. If 'none' specified, no archiving will occur.",
+                                choices=["none", "zip", "gzip", "bz2", "xz"], default="none")
+        parser.add_argument('-code', '--should-build-code', default=True,action='store_true', help='Toggles building all code for the project by launcher type (game, server, unified).')
+        parser.add_argument('-assets', '--should-build-assets', default=True, action='store_true', help='Toggles building all assets for the project by launcher type (game, server, unified).')
+        parser.add_argument('-sl', '--seedlist', type=pathlib.Path, dest='seedlist_paths', action='append',
+                                help='Path to a seed list file for asset bundling. Specify multiple times for each seed list.')
+        parser.add_argument('-gf', '--gem-folder-name-to-copy', type=str, dest='gem_folder_names_to_copy', action='append',
+                                help="The name of a Gem folder to include in release. Only specify folder name."
+                                    "This should match what is in the project directory.")
+        parser.add_argument('-ggf', '--game-gem-folder-name-to-copy', type=str, dest='game_gem_folder_names_to_copy', action='append',
+                                help="The name of a Gem folder to include in release for a Game Package. Only specify folder name."
+                                    "This should match what is in the project directory.")
+        parser.add_argument('-sgf', '--server-gem-folder-name-to-copy', type=str, dest='server_gem_folder_names_to_copy', action='append',
+                                help="The name of a Gem folder to include in release for a Server Package. Only specify folder name."
+                                    "This should match what is in the project directory.")
+        parser.add_argument('-gpfp', '--game-project-file-pattern-to-copy', type=str, dest='game_project_file_patterns_to_copy', action='append',
+                                help="Any additional file patterns located in the project directory. File patterns will be relative to the project path. Will be included in GameLauncher.")
```

Comment:
Terminology wise I think we should use the term `client` over `game`, since it pairs with the term `server` and is also applicable for content that is not a game (such as a running a simulation).

We can't change the `${project}.GameLauncher` name currently, but we can use the correct terminology going forward.

Any Thoughts @tkothadev @AMZN-alexpete @mbalfour-amzn @AMZN-Olex @gene-walt @spham-amzn ?

Reply:
I could see the use case with a networked game like MPS, but with an offline game, that terminology won't make sense. Game is probably the closest compromise implying client nature, which is why I suspect that stuck.

One counter point though to my argument there is that you could just use the general `gem-folder-name` or `project-pattern-file`, and prevent building server via `-noserver`, so all general file parameters only go to the game launcher. 

That would get the same effect, but it doesn't erase the fact that `ClientPackage` for a singleplayer game implies networking where there is none, imo.

Reply:
I think Game Launcher is fine, since that works for single player games just fine. It's not necessarily a client. In fact, if you link Game Launcher with Multiplayer.Unified.Static it can serve as a host not just a client.

Reply:
I agree, `client` should be use in the terminology. I think there should be a deprecation path for `.GameLauncher` in the future, since O3DE can be used for simulations as well, in which the term `Game` would be confusing or misleading.

Reply:
For now I will defer this discussion for a future PR. I will make an issue of this for future reference.

Reply:
> For now I will defer this discussion for a future PR. I will make an issue of this for future reference.

I am good with that decision